backend/docker_ops.py

```
@@ -157,6 +177,38 @@ async def start_compose_stack(self, compose_dir: str) -> bool:
         logger.info(f"Successfully started compose stack in {compose_dir}")
         return True
     
+    async def start_compose_stack_remote(self, target_host: str, compose_dir: str, 
+                                       ssh_user: str = "root", ssh_port: int = 22) -> bool:
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)